docs/cues/cue-playback.md

```diff
@@ -86,33 +86,45 @@ graphic you can jump to a page number by entering the number.
 
 >   If you have stored cues on the touch select buttons, you select
     different pages using the Playbacks page touch buttons.
+    
+---
+
+The way how playbacks behave when pages are changed while playbacks are active
+can be set with the [\[Playback Paging\] user setting](../system-settings/user-settings.md#playback-paging).
 
 ### Consoles without Motorised Faders
 
--   Playbacks which are fired when you change page remain active. If you
+-   By default, playbacks which are fired when you change page remain active. If you
     want to fire a cue on a fader which is already up from a previous
     page, lower the fader to zero then raise it again. The cue from the
     previous page will stop and the cue from the new page will fire.
+    This setting is called \[Always Hold\].
 
--   If you return to a page with an active playback, the fader will not
+-   If changed to \[Never Hold\] then the behaviour allows for active faders on
+    multiple pages: if you return to a page with an active playback, the fader will not
     resume control of the playback until it matches the current level of
     the playback. This prevents the playback level jumping when the
     fader is first moved. If a playback is active from another page the
     display area is purple and shows the page number at the top in light
-    blue. You can change this behaviour using the 
-    [\[Playback Paging\] user setting](../system-settings/user-settings.md#playback-paging).
+    blue. 
+
+The setting \[Normal\] gives the default state, for consoles 
+without motorised faders this is \[Always Hold\].
 
 ### Consoles with Motorised Faders
 
--   Playbacks which are fired when you change page remain active, but
+-   By default, playbacks which are fired when you change page remain active, but
```

Comment:
I think that the playback will always remain active? What changes is whether it pages?

Reply:
Yes that is now a bit confusing. I will rewrite this section again to make that clearer